public/css/main.css

```diff
@@ -37,13 +37,14 @@ body {
   display: flex;
   justify-content: space-between;
   align-items: center;
-  box-shadow: 0 .2rem .2rem #DADADA;
+  box-shadow: 0 .1rem .3rem;
 }
 
 .naz-events {
   color: white;
   font-weight: bold;
   font-size: 2.3rem;
+  margin: auto;
```

Comment:
I see you did this because there's no calendar on the event details page, but I don't see any reason not to include the calendar here - what do you think? Add it in?